docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -1,213 +1,185 @@
+```eval_rst
+.. important::
 
-        .. seealso::
+   ``bigdl-llm`` is tested with Python 3.9, which is recommended for best practices.
+```
 
-           We recommend you to use `this offline package <https://registrationcenter-download.intel.com/akdlm/IRC_NAS/20f4e6a1-6b0b-4752-b8c1-e5eacba10e01/l_BaseKit_p_2024.0.0.49564_offline.sh>`_ to install oneapi.
+The easiest ways to install `bigdl-llm` is the following commands:
 
-   .. tab:: Windows
+```
+conda create -n llm python=3.9 libuv
+conda activate llm
 
-      BigDL-LLM on Windows supports Intel iGPU and dGPU.
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+```
 
-      To apply Intel GPU acceleration, there're several prerequisite steps for tools installation and environment preparation:
+### Install BigDL-LLM From Wheel
 
-      * Step 1: Install `Visual Studio 2022 <https://visualstudio.microsoft.com/downloads/>`_ Community Edition and select "Desktop development with C++" workload
+If you encounter network issues when installing IPEX, you can also install BigDL-LLM dependencies for Intel XPU from source achieves. First you need to download and install torch/torchvision/ipex from wheels listed here before installing `bigdl-llm`.
 
-      * Step 2: Install or update to latest GPU driver
+Download the wheels on Windows system:
 
-      * Step 3: Install `Intel® oneAPI Base Toolkit <https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html>`_ 2024.0
+```
+wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torch-2.1.0a0%2Bcxx11.abi-cp39-cp39-win_amd64.whl
+wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/torchvision-0.16.0a0%2Bcxx11.abi-cp39-cp39-win_amd64.whl
+wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/xpu/intel_extension_for_pytorch-2.1.10%2Bxpu-cp39-cp39-win_amd64.whl
 ```
 
-### Install BigDL-LLM
-
-We recommend using [Conda](https://docs.conda.io/en/latest/miniconda.html) to create a python 3.9 enviroment:
-
-```eval_rst
-.. important::
+You may install dependencies directly from the wheel archives and then install `bigdl-llm` using following commands:
 
-   ``bigdl-llm`` is tested with Python 3.9, which is recommended for best practices.
 ```
+pip install torch-2.1.0a0+cxx11.abi-cp39-cp39-win_amd64.whl
+pip install torchvision-0.16.0a0+cxx11.abi-cp39-cp39-win_amd64.whl
+pip install intel_extension_for_pytorch-2.1.10+xpu-cp39-cp39-win_amd64.whl
 
-```eval_rst
-.. tabs::
-   .. tab:: Linux
-
-      .. code-block:: bash
+pip install --pre --upgrade bigdl-llm[xpu]
+```
 
-         conda create -n llm python=3.9
-         conda activate llm
+### Runtime Configuration
 
-         pip install --pre --upgrade bigdl-llm[xpu_2.1] -f https://developer.intel.com/ipex-whl-stable-xpu
+To use GPU acceleration on Windows, several environment variables are required or recommended before running a GPU example.
```

Comment:
If `SYCL_CACHE_PERSISTENT=1` and `set BIGDL_LLM_XMX_DISABLED=1` are all necessary to do inference, change here to `several environment variables are required before running a GPU example.`
If not, maybe document which are required, which are recommended for performance.

Reply:
yes, they are all necessary